app/view_models/nsm/v1/core_cost_summary.rb

```diff
@@ -86,12 +90,22 @@ def calculate_disbursements(formatted:)
         )
       end
 
-      def calculate_row(rows, name, formatted:)
-        net_cost = rows.sum(&:provider_requested_amount)
-        allowed_net_cost = show_allowed? ? rows.sum(&:caseworker_amount) : nil
+      def calculate_work_items_row(type, name, formatted:, extra_rows: [])
+        claimed_rows = work_items_of_type(type, type_type: :claimed)
+        net_cost = claimed_rows.sum(&:provider_requested_amount) + extra_rows.sum(&:provider_requested_amount)
+        if show_allowed?
+          allowed_rows = work_items_of_type(type, type_type: :allowed)
+          allowed_net_cost = allowed_rows.sum(&:caseworker_amount) + extra_rows.sum(&:caseworker_amount)
+        end
+
+        calculate_hash(name, claimed_rows, allowed_rows, net_cost, allowed_net_cost, formatted)
+      end
+
+      def calculate_hash(name, claimed_rows, allowed_rows, net_cost, allowed_net_cost, formatted)
+        any_changed_type = show_allowed? && claimed_rows.any? { !_1.in?(allowed_rows) }
```

Comment:
Since you're only using `allowed_rows` here, wouldn't it be simpler to just calculate this bool outside and pass _that_ in? `claimed_rows` seems unused after too

Reply:
Fair